.github/workflows/verify.yml

```diff
@@ -12,8 +12,6 @@ jobs:
     steps:
       - name: Clone repository
         uses: actions/checkout@v3
-        with:
-          token: ${{ secrets.WAGMI_BOT_GITHUB_TOKEN }}
```

Comment:
Does this not need to be set? Is it okay to remove without other changes?

Reply:
@jrhender IMO we don't need any token to clone the repo, might be something that devs from `wagmi` have missed. This was corrected by @Timtech4u.